build/prebuild.sh

```diff
@@ -9,7 +9,7 @@ TARGETDIR='node_modules/grafana_master'
 
 if [ ! -d $TARGETDIR ]; then
   echo "Cloning Grafana source into $TARGETDIR"
-  git clone https://github.com/grafana/grafana.git --depth 1 $TARGETDIR
+  git clone https://github.com/grafana/grafana.git --single-branch --branch v5.4.3 --depth 1 $TARGETDIR
```

Comment:
why an old version? Why does this need grafana?

Reply:
It uses SASS variables from Grafana
Old version is needed because Grafana 6.x uses `public/sass/_variables.generated.scss` instead of `public/sass/_variables.scss` in 5.x